doc/manuals/user/forbidden_characters.md

```diff
@@ -46,6 +46,13 @@ attribute identifiers as part of the URL, their use should be avoided.
 The use of these characters is perfectly safe when only standard operations
 are involved, anyway. 
 
+### Exceptions
+
+There are some exception cases in which the above restrictions are not checked. In particuler, in the following fields:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/1807/files#diff-7ed462fcb6ea86180aabc505a3c08a41